extensions/wikia/CommunityPage/templates/recentlyJoined.mustache

```diff
@@ -1,11 +1,15 @@
-{{#haveMembers}}
-	<h2>{{recentlyJoinedHeaderText}}</h2>
-	{{#members}}
-		<div class="avatar-container">
-			<a href="{{profilePage}}">{{{avatar}}}</a>
+<div class="community-page-recently-joined">
+	{{#haveMembers}}
+		<div class="members">
+			<h2>{{recentlyJoinedHeaderText}}</h2>
+			{{#members}}
+				<div class="avatar-container">
+					<a href="{{profilePage}}">{{{avatar}}}</a>
+				</div>
+			{{/members}}
 		</div>
-	{{/members}}
-{{/haveMembers}}
-<span class="recently-joined">
-	<a href="#" id="viewAllMembers">{{allMembers}}</a>
-</span>
+	{{/haveMembers}}
+	<span class="more-link">
```

Comment:
It turns out that haveMembers checks only for recently joined members so I'm sure viewAllMembers link is outside condition on purpose. Renamed haveMembers to haveNewMembers to better reflect it's meaning.